process/superconductors.py

```diff
@@ -10,14 +10,26 @@
 logger = logging.getLogger(__name__)
 
 
-def jcrit_rebco(temperature, b):
-    """Critical current density for "REBCO" 2nd generation HTS superconductor
-    temperature : input real : superconductor temperature (K)
-    b : input real : Magnetic field at superconductor (T)
-    jcrit : output real : Critical current density in superconductor (A/m2)
-
-    Will return a negative number if the temperature is greater than Tc0, the
-    zero-field critical temperature.
+def jcrit_rebco(temperature: float, b: float) -> tuple[float, bool]:
+    """
+    Critical current density for "REBCO" 2nd generation HTS superconductor.
+
+    :param temperature: Superconductor temperature (K).
+    :type temperature: float
```

Comment:
Is this not what the style guide suggests? Not sure I like it mind you but is this not what it shows as an example for a function?

```suggestion
    :param float temperature: Superconductor temperature (K).
```